.gitignore

```diff
@@ -26,7 +26,6 @@ doc/_build
 doc/source
 
 *.pyccel
-*.f90
```

Comment:
Why should we ignore all Fortran files? If they are in `__pyccel__` they are ignored already.

Reply:
Presumably this line predated the `__pyccel__` folder? In any case I agree that it can be removed